### PR TITLE
Introducing JSX translation

### PR DESCRIPTION
**Pull request by gergely-nagy**

### CODE REVIEW

File: content/docs/introducing-jsx.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
+JSX-nek hívják, ami egy szintaxis kiterjesztés a JavaScript nyelvhez. Használatát ajánljuk a React-el, hogy leírd miként fog kinézni a felhasználói felület. A JSX egy sablonnyelvre emlékeztethet, de a JavaScript teljes erejével bír.
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+A JSX React "elemeket" állít elő. A [következő szekcióban](/docs/rendering-elements.html) fogjuk megvizsgálni a DOM-ba való renderelésüket. Lent megtalálod a JSX alapjait, amelyek az elinduláshoz szükségesek.
 
-### Why JSX? {#why-jsx}
+### Miért a JSX? {#why-jsx}
 
-React embraces the fact that rendering logic is inherently coupled with other UI logic: how events are handled, how the state changes over time, and how the data is prepared for display.
+A React felöleli a tényt, hogy a megjelenítési logika eleve összekapcsolódik a többi felhasználói felület logikájával: hogyan kezelik az eseményeket, hogyan változnak az állapotok az idő múlásával, és hogyan állnak elő az adatokat a megjelenítéshez.
```
Comment (balazsorban44) [inline content/docs/introducing-jsx.md]:
```suggestion
A React azt az elvet vallja, hogy a megjelenítési logika eleve összekapcsolódik a többi felhasználói felületi logikával: az események kezelésével, állapotok időbeni változásával, és hogy az adat hogyan van előkészítve annak megjelenítéséhez.
```

I am not sure about this one though.

File: content/docs/introducing-jsx.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
 
-Instead of artificially separating *technologies* by putting markup and logic in separate files, React [separates *concerns*](https://en.wikipedia.org/wiki/Separation_of_concerns) with loosely coupled units called "components" that contain both. We will come back to components in a [further section](/docs/components-and-props.html), but if you're not yet comfortable putting markup in JS, [this talk](https://www.youtube.com/watch?v=x7cQ3mrcKaY) might convince you otherwise.
+Ahelyett, hogy a *technológiákat* mesterségesen elválasztaná azzal, hogy a jelölést és a logikát külön fájlokba helyezi, a React szempontok szerint szeparál lazán összekapcsolt „komponenseknek” nevezett egységekkel, amelyek mindkettőt tartalmazzák. Vissza fogunk térni a komponensekhez egy [későbbi szekcióban](/docs/components-and-props.html), ha még nem vagy otthon JS-be a jelölés használatával, [ez a beszélgetés](https://www.youtube.com/watch?v=x7cQ3mrcKaY) meggyőzhet téged.
```
Comment (balazsorban44) [inline content/docs/introducing-jsx.md]:
```suggestion
Ahelyett, hogy a *technológiákat* mesterségesen elválasztaná azzal, hogy a jelölést és a logikát külön fájlokba helyezi, a React szempontok szerint szeparál lazán összekapcsolt „komponenseknek” nevezett egységekkel, amelyek mindkettőt tartalmazzák. Vissza fogunk térni a komponensekhez egy [későbbi szekcióban](/docs/components-and-props.html), ha még nem vagy otthon a jelölés JS-ben való használatával, [ez a beszélgetés](https://www.youtube.com/watch?v=x7cQ3mrcKaY) meggyőzhet téged.
```

File: content/docs/introducing-jsx.md [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
+A React [nem követeli meg](/docs/react-without-jsx.html) a JSX használatát, de a legtöbb ember hasznosnak találja mint vizuális segédeszköz ha felhasználói felülettel dolgozik JavaScript kódban. Ezenkívül azt is lehetővé teszi, hogy a React hasznosabb hiba- és figyelmeztető üzeneteket jelenítsen meg.
 
-With that out of the way, let's get started!
+Ezeket tisztázva, kezdjünk is bele!
 
-### Embedding Expressions in JSX {#embedding-expressions-in-jsx}
+### Beágyazott kifejezések JSX-ben {#embedding-expressions-in-jsx}
 
-In the example below, we declare a variable called `name` and then use it inside JSX by wrapping it in curly braces:
+A következő példában egy `name` nevű változót deklarálunk, majd a JSX-ben használjuk úgy, hogy kapcsos zárójelek közé csomagoljuk:
```
Comment (balazsorban44) [inline content/docs/introducing-jsx.md]:
```suggestion
A következő példában egy `name` változót deklarálunk, majd a JSX-ben használjuk úgy, hogy kapcsos zárójelbe tesszük:
```

File: content/docs/introducing-jsx.md [hunk 4]
```diff
@@ -1,50 +1,50 @@
...
+const element = <h1>Helló, {name}</h1>;
 
 ReactDOM.render(
   element,
   document.getElementById('root')
 );
 ```
 
-You can put any valid [JavaScript expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Expressions) inside the curly braces in JSX. For example, `2 + 2`, `user.firstName`, or `formatName(user)` are all valid JavaScript expressions.
+A JSX-be bármilyen [JavaScript kifejezést](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Expressions) beilleszthetsz kapcsos zárójelek közé. Például a `2 + 2`, `user.firstName`, vagy a `formatName(user)` mind érvényes JavaScript kifejezések.
```
Comment (balazsorban44) [inline content/docs/introducing-jsx.md]:
```suggestion
A JSX-be bármilyen érvényes [JavaScript kifejezést](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Expressions) beilleszthetsz kapcsos zárójelek közé. Például a `2 + 2`, `user.firstName`, vagy a `formatName(user)` mind érvényes JavaScript kifejezések.
```

File: content/docs/introducing-jsx.md [hunk 5]
```diff
@@ -70,86 +70,86 @@ ReactDOM.render(
 
 [](codepen://introducing-jsx)
 
-We split JSX over multiple lines for readability. While it isn't required, when doing this, we also recommend wrapping it in parentheses to avoid the pitfalls of [automatic semicolon insertion](https://stackoverflow.com/q/2846283).
+A JSX-et több sorra bontottuk az olvashatóság érdekében. Bár ez nem kötelező, de ha ezt csinálod akkor javasoljuk, hogy csomagold zárójelek közé, elkerülve az [automatikus pontosvessző beillesztésének](https://stackoverflow.com/q/2846283) buktatóit.
```
Comment (balazsorban44) [inline content/docs/introducing-jsx.md]:
```suggestion
A JSX-et több sorra bontottuk az olvashatóság érdekében. Bár ez nem kötelező, de ha így csinálod akkor azt javasoljuk, hogy tedd zárójelek közé, hogy elkerüld az [automatikus pontosvessző beillesztésének](https://stackoverflow.com/q/2846283) buktatóit.
```

File: content/docs/introducing-jsx.md [hunk 6]
```diff
@@ -70,86 +70,86 @@ ReactDOM.render(
...
+    <h2>Jó itt látni.</h2>
   </div>
 );
 ```
 
-### JSX Prevents Injection Attacks {#jsx-prevents-injection-attacks}
+### A JSX megakadályozza az injekciós támadásokat {#jsx-prevents-injection-attacks}
 
-It is safe to embed user input in JSX:
+Biztonságos a felhasználói bevitel beágyazása a JSX-be:
```
Comment (balazsorban44) [inline content/docs/introducing-jsx.md]:
```suggestion
A felhasználói bevitel beágyazása a JSX-be biztonságos:
```

File: content/docs/introducing-jsx.md [hunk 7]
```diff
@@ -70,86 +70,86 @@ ReactDOM.render(
...
 
 ```js
 const title = response.potentiallyMaliciousInput;
-// This is safe:
+// Ez biztonságos:
 const element = <h1>{title}</h1>;
 ```
 
-By default, React DOM [escapes](https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html) any values embedded in JSX before rendering them. Thus it ensures that you can never inject anything that's not explicitly written in your application. Everything is converted to a string before being rendered. This helps prevent [XSS (cross-site-scripting)](https://en.wikipedia.org/wiki/Cross-site_scripting) attacks.
+Alapértelmezés szerint a React DOM [elkerüli](https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html) az összes JSX-be beágyazott értéket, mielőtt azokat megjelenítené. Így tudja biztosítani, hogy soha nem tudsz bármi olyat injektálni, amit kifejezetten nem a te alkalmazásodban írtak. Minden átalakul karakterlánccá, mielőtt megjelenítésre kerül. Ez segít megelőzni az [XSS (cross-site-scripting)](https://en.wikipedia.org/wiki/Cross-site_scripting) típusú támadásokat.
```
Comment (balazsorban44) [inline content/docs/introducing-jsx.md]:
```suggestion
Alapértelmezés szerint a React DOM [szanitálja](https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html) az összes JSX-be beágyazott értéket, mielőtt azokat megjelenítené. Így biztosítja azt, hogy soha nem tudsz bármi olyat injektálni, amit kifejezetten nem a te alkalmazásodban írtak. Minden átalakul karakterlánccá, mielőtt megjelenítésre kerül. Ez segít megelőzni az [XSS (cross-site-scripting)](https://en.wikipedia.org/wiki/Cross-site_scripting) típusú támadásokat.
```

File: content/docs/introducing-jsx.md (hunk 2 above)
Comment (balazsorban44) [reply content/docs/introducing-jsx.md]: @gergely-nagy forgot this one :)

**Comment by gergely-nagy** [comment]
Oh, sorry, I missed the last one.

**Pull request merged**